index.html

```diff
@@ -4,12 +4,546 @@
+                    </div>
+                </div>
+                <div class="section1-grid article2">
+                    <div>
+                        <img class="writer" src="images/photo.jpg.png" alt="writer">
+                        <span class="author">Author</span>
+                    </div>
+                    <p class="section1-grid-para">
+                        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nam sed perspiciatis minus consequatur
+                        ?
+                    </p>
+                    <div>
+                        <p class="comments">
+                            <i class="fas fa-comment-alt" aria-hidden="true"></i> "3 comments"
+                        </p>
+                    </div>
+                    <div class="article-tag flex">
+                        <p># React</p>
+                        <p># Javascript</p>
+                        <p># Frameworks</p>
+                    </div>
+                </div>
+                <div class="section1-grid article3">
+                    <div>
+                        <img class="writer" src="images/photo.jpg.png" alt="writer" >
+                        <span class="author">Author</span>
+                    </div>
+                    <p class="section1-grid-para">
+                        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nam sed perspiciatis minus consequatur
+                        ?
+                    </p>
+                    <div>
+                        <p class="comments">
+                            <i class="fas fa-comment-alt" aria-hidden="true"></i> "3 comments"
+                        </p>
+                    </div>
+                    <div class="article-tag flex">
+                        <p># React</p>
+                        <p># Javascript</p>
+                        <p># Frameworks</p>
+                    </div>
+                </div>
+                <div class="section1-grid article4">
+                    <div>
+                        <img class="writer" src="images/photo.jpg.png" alt="writer" >
+                        <span class="author">Author</span>
+                    </div>
+                    <p class="section1-grid-para">
+                        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nam sed perspiciatis minus consequatur
+                        ?
+                    </p>
+                    <div>
+                        <p class="comments">
+                            <i class="fas fa-comment-alt" aria-hidden="true"></i> "3 comments"
+                        </p>
+                    </div>
+                    <div class="article-tag flex">
+                        <p># React</p>
+                        <p># Javascript</p>
+                        <p># Frameworks</p>
+                    </div>
+                </div>
+            </section>
+            <div class="section1-head">Don’t Miss These Latest Articles
+            </div>
```

Comment:
- [ ] this heatmap is supposed to be compatible with the 1440px version of the smashing magazine, "don't miss these latest articles" isn't centered on that size
![image](https://user-images.githubusercontent.com/11758151/92989756-76b16900-f4ce-11ea-9599-15a3f71eaa47.png)